### ISSUE TITLE
Status of testing of Apache Airflow 2.2.1rc2

### ISSUE THREAD

**Issue by potiuk** [apache/airflow#19246]
### Airflow 2.2.1rc2 testing

We have a kind request for all the contributors to the latest [Apache Airflow RC 2.2.1rc2](https://pypi.org/project/apache-airflow/2.2.1rc2/).

Could you please help us to test the RC versions of Airflow?

Please let us know in the comment if the issue is addressed in the latest RC.

- [x] [Document changes related to PR #16634 (#18644)](https://github.com/apache/airflow/pull/18644): @Jorricks
- [ ] [Don't install SQLAlchemy/Pendulum adapters for other DBs (#18745)](https://github.com/apache/airflow/pull/18745): @JavierLopezT @ashb
     Linked issues:
     - [ImportError: /usr/lib/x86_64-linux-gnu/libstdc++.so.6: cannot allocate memory in static TLS block (#17546)](https://github.com/apache/airflow/issues/17546)
- [ ] [Don't bake ENV and _cmd into tmp config for non-sudo (#18772)](https://github.com/apache/airflow/pull/18772): @ashb @khalidmammadov
     Linked issues:
     - [Remove duplicate line of code for configuration copy (#18723)](https://github.com/apache/airflow/pull/18723)
- [x] [Add Callbacks Section to Logging & Monitoring (#18842)](https://github.com/apache/airflow/pull/18842): @john-jac
- [x] [Add information about keepalives for managed Postgres (#18850)](https://github.com/apache/airflow/pull/18850): @hpatel-higi @potiuk
     Linked issues:
     - [Scheduler stops running: psycopg2.OperationalError: SSL SYSCALL error: EOF detected (#18846)](https://github.com/apache/airflow/issues/18846)
- [x] [Fix occassional deadlock on MSSQL test DagMaker cleanup (#18857)](https://github.com/apache/airflow/pull/18857): @potiuk
- [x] [Group PATCH DAGrun together with other DAGRun endpoints (#18885)](https://github.com/apache/airflow/pull/18885): @BasPH
- [x] [Fix catchup by limiting queued dagrun creation using max_active_runs (#18897)](https://github.com/apache/airflow/pull/18897): @ephraimbuddy
- [x] [Check python version before starting triggerer (#18926)](https://github.com/apache/airflow/pull/18926): @jedcunningham
- [ ] [Try to move "dangling" rows in upgradedb (#18953)](https://github.com/apache/airflow/pull/18953): @uranusjr @sbialkowski-pixel
     Linked issues:
     - [Migrate from 2.1.4 to 2.2.0 (#18894)](https://github.com/apache/airflow/issues/18894)
- [x] [Fix XCom.delete error in Airflow 2.2.0 (#18956)](https://github.com/apache/airflow/pull/18956): @jordanjeremy
- [x] [Row lock TI query in SchedulerJob._process_executor_events (#18975)](https://github.com/apache/airflow/pull/18975): @ephraimbuddy
- [x] [Fixes #18943 null execution date on insert to task_fail violating NOT NULL (#18979)](https://github.com/apache/airflow/pull/18979): @robinedwards
     Linked issues:
     - [IntegrityError inserting into task_fail table with null execution_date from TI.handle_failure_with_callback (#18943)](https://github.com/apache/airflow/issues/18943)
- [x] [Sentry before send fallback (#18980)](https://github.com/apache/airflow/pull/18980): @leonsmith @joshzana
     Linked issues:
     - [Missing value for new before_send config causes airflow.exceptions.AirflowConfigException: section/key [sentry/before_send] not found in config (#18963)](https://github.com/apache/airflow/issues/18963)
- [ ] [Upgrade old DAG/task param format when deserializing from the DB (#18986)](https://github.com/apache/airflow/pull/18986): @ashb
- [ ] [fail backfill command before loading DAGs if missing args (#18994)](https://github.com/apache/airflow/pull/18994): @SamWheating
- [x] [Workaround libstdcpp TLS error (#19010)](https://github.com/apache/airflow/pull/19010): @potiuk
- [x] [Allow Param to support a default value of ``None`` (#19034)](https://github.com/apache/airflow/pull/19034): @jedcunningham
- [ ] [Warn about unsupported Python 3.10 (#19060)](https://github.com/apache/airflow/pull/19060): @mik-laj
- [x] [Rename trigger page label to Logical Date (#19061)](https://github.com/apache/airflow/pull/19061): @uranusjr
- [x] [Remove incorrect type comment in Swagger2Specification._set_defaults classmethod (#19065)](https://github.com/apache/airflow/pull/19065): @brain-buster
- [x] [Relax packaging requirement (#19087)](https://github.com/apache/airflow/pull/19087): @uranusjr @andrewgodwin
     Linked issues:
     - [Add "packaging" to core requirements (#18122)](https://github.com/apache/airflow/pull/18122)
- [ ] [Change `ds`, `ts`, etc. back to use logical date (#19088)](https://github.com/apache/airflow/pull/19088): @uranusjr
- [ ] [Ensure task state doesn't change when marked as failed/success/skipped (#19095)](https://github.com/apache/airflow/pull/19095): @ephraimbuddy
- [ ] [add detailed information to logging when a dag or a task finishes. (#19097)](https://github.com/apache/airflow/pull/19097): @steveyz-astro
- [ ] [Hide tooltip when next run is none (#19112)](https://github.com/apache/airflow/pull/19112): @bbovenzi
- [x] [Prevent scheduler crash when serialized dag is missing (#19113)](https://github.com/apache/airflow/pull/19113): @ephraimbuddy
- [ ] [Fix queued dag runs changes catchup=False behaviour (#19130)](https://github.com/apache/airflow/pull/19130): @robinedwards
- [ ] [Add test for interval timetable catchup=False (#19145)](https://github.com/apache/airflow/pull/19145): @uranusjr @robinedwards
     Linked issues:
     - [Fix queued dag runs changes catchup=False behaviour (#19130)](https://github.com/apache/airflow/pull/19130)
- [ ] [Crerate TI context with data interval compat layer (#19148)](https://github.com/apache/airflow/pull/19148): @uranusjr @andreychernih
     Linked issues:
     - [TypeError: unsupported operand type(s) for +=: 'NoneType' and 'datetime.timedelta' (#19143)](https://github.com/apache/airflow/issues/19143)
- [x] [Faster db migration to Airflow 2.2 (#19166)](https://github.com/apache/airflow/pull/19166): @jedcunningham
- [x] [Add TriggererJob to jobs check command (#19179)](https://github.com/apache/airflow/pull/19179): @mik-laj
- [x] [Clearing ti.next_method and ti.next_kwargs on task finish. (#19183)](https://github.com/apache/airflow/pull/19183): @ReadytoRocc @jedcunningham
     Linked issues:
     - [Deferred TI's `next_method` is not cleared when Clearing a Successful Task (#19120)](https://github.com/apache/airflow/issues/19120)
     - [Deferred TI's `next_method` and `next_kwargs` not cleared on retries (#18146)](https://github.com/apache/airflow/issues/18146)
- [x] [Fix airflow jobs check cmd for TriggererJob (#19185)](https://github.com/apache/airflow/pull/19185): @mik-laj
- [x] [Add DagRun.logical_date as a property (#19198)](https://github.com/apache/airflow/pull/19198): @uranusjr
- [x] [Fix Unexpected commit error in schedulerjob (#19213)](https://github.com/apache/airflow/pull/19213): @ephraimbuddy
     Linked issues:
     - [Ensure task state doesn't change when marked as failed/success/skipped (#19095)](https://github.com/apache/airflow/pull/19095)

Thanks to all who contributed to the release (probably not a complete list!):
@ephraimbuddy @JavierLopezT @jedcunningham @sbialkowski-pixel @Jorricks @khalidmammadov @john-jac @brain-buster @andrewgodwin @joshzana @SamWheating @mik-laj @leonsmith @ReadytoRocc @ashb @robinedwards @potiuk @bbovenzi @jordanjeremy @steveyz-astro @BasPH @hpatel-higi @uranusjr @andreychernih

### Committer

- [X] I acknowledge that I am a maintainer/committer of the Apache Airflow project.
